### ISSUE TITLE
Will you be packaging this up?

### ISSUE THREAD

**Issue by swythan** [howardyoo/airflow_otel_provider#5]
Hi again. Do you have any plans for packaging this up in the near future so it can be easily installed without cloning the repo? 

As mentioned in #3 this library (and the tracing support you added to Airflow itself) have come along at a great time for my company (G-Research). We have a dedicated OSS team that I can probably get to help if you need a hand with anything.

**Comment by howardyoo** [comment]
> Hi again. Do you have any plans for packaging this up in the near future so it can be easily installed without cloning the repo?
> 
> As mentioned in #3 this library (and the tracing support you added to Airflow itself) have come along at a great time for my company (G-Research). We have a dedicated OSS team that I can probably get to help if you need a hand with anything.

Yes, that was actually the original plan. However, due to the commiters getting quite busy for airflow 3.0 related works, the propose that I submitted (for making otel provider as part of official providers) did not get the adequate amount of vote, so it slipped (and I was recommended to add it to the airflow ecosystem page : https://airflow.apache.org/ecosystem/ )

So, not in the near future, but certainly something that can be doable in future, if there are adequate amount of interests.

**Comment by swythan** [comment]
Thanks, Howard. I've [commented](https://github.com/apache/airflow/issues/37628#issuecomment-2340040799) on the issue on the Airflow side to add some weight to it.
